tools/masspix/macros.xml

```diff
@@ -0,0 +1,36 @@
+<macros>
+  <xml name="requirements">
+    <requirements>
+      <requirement type="package" version="1.6.6">r-optparse</requirement>
+      <requirement type="package" version="1.7.7">r-calibrate</requirement>
+      <requirement type="package" version="0.9_13">r-rjava</requirement>
+      <requirement type="package" version="8.0.112">java-jdk</requirement>
```

Comment:
If openjdk not working? java-jdk is really old package - means not maintained afaik for conda. I think if we can we really should openjdk.

Reply:
Let me test openjdk again.